bindata/network-diagnostics/network-check-target.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: network-check-target
+  namespace: openshift-network-diagnostics
+  annotations:
+    kubernetes.io/description: |
+      This daemonset deploys the network-check-target pods that run
+      a dummy hello-openshift app to be checked by network-check-source pod
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  selector:
+      matchLabels:
+        app: network-check-target
+  template:
+    metadata:
+      labels:
+        app: network-check-target
+    spec:
+      containers:
+      - image: "{{.NetworkCheckTargetImage}}"
+        imagePullPolicy: IfNotPresent
+        name: network-check-target-container
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+        resources:
+          requests:
+            cpu: 10m
+            memory: 150Mi
+        readinessProbe:
+          httpGet:
+            port: 8080
+            path: /
+          initialDelaySeconds: 30
+          timeoutSeconds: 10
+      serviceAccount: default
+      terminationGracePeriodSeconds: 10
+      tolerations:
```

Comment:
I don't think we want to tolerate anything. If a node is tainted, we shouldn't deploy on to it.

Reply:
fair, this I cargo culted, will amend.

Reply:
done

Reply:
Actually ,I take that back.
I need the toleration, as I want the hello-openshift to be deployed on masters too, so the source pod can do a network check against the whole cluster.
I removed it from network-check-source tho, as that I don't care which node is.

Reply:
Hmm. I suppose we don't have a better answer than tolerating all taints. Bah.